Formula/c/c3c.rb

```diff
@@ -2,7 +2,7 @@ class C3c < Formula
   desc "Compiler for the C3 language"
   homepage "https://github.com/c3lang/c3c"
   url "https://github.com/c3lang/c3c/archive/refs/tags/v0.7.4.tar.gz"
-  sha256 "65bd76c5b418382ccccde468d5cc0a1331ca3bad27f1e1828a43f078d87dcdc0"
+  sha256 "410267a3d771ac4b4d6bcc29be0faf30d4959c24b31f9d1bd00661656072dc2b"
   license "LGPL-3.0-only"
   head "https://github.com/c3lang/c3c.git", branch: "master"
 
```

Comment:
Looks like I forgot a revision bump since linkage will be broken after LLVM 21 PR.